cms_status_message_example/templates/test_page.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+  <template id="cms_status_message_example.test_form_page" name="CMS status test page">
+
+    <t t-call="website.layout">
+      <form method="get">
+        <div>
+          <label for="name">Message</label>
```

Comment:
`for="message"`

Reply:
I'd add something like this to explain: `<p>Enter a message and submit the form. You'll see your message rendered as a status message.</p>`